pygame/image.py

```diff
@@ -49,13 +51,14 @@ def save(surface, filename):
     fn_normalized = filename.lower()
     result = 0
     # TODO: prep/unprep surface
-    if fn_normalized.endswith('bmp'):
+    if fn_normalized.endswith(as_bytes('bmp')):
```

Comment:
Do we really need as_bytes here?

Don't we only support recent enough python 2 versions that we can just use b'bmp'?